lib/message/encrypt.js

```diff
@@ -16,6 +16,7 @@ export default async function encryptMessage({ armor = true, ...options }) {
     options.compression = options.compression ? openpgp.enums.compression.zlib : undefined;
 
     options.armor = armor;
+    options.returnSessionKey = returnSessionKey;
```

Comment:
Instead of making it a default and setting it here, I would just set `options.returnSessionKey = true`, but below instead of here (inside the `if (options.detached) {`, since that's when you need it).